imageresizerapp/imageresizerapp.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>imageresizerapp</id>
```

Comment:
It's not too late. You can rename the `nuspec` file and `id` value and publish, provided the new name is available. I could unlist this version or make it a virtual package (dependent only on the new package) or both.

I can't find the thread now, but I remember a moderator insisting on the name imageresizer**app**. This package was first published in early 2014, back when the naming-meta wasn't well defined.

I wouldn't use the `.install` unless there were also a `.portable` edition of this package (usually a ZIP archive).

I would encourage you to try simply, `imageresizer`. I think the naming-meta has caught up and that should be fine! I would permit it, in any case.